File: solutions/jose-rivas/python/p-02-arrays/exercise-01.py
try:
    N = int(input("Entrez le nombre d'elements : "))

    if N < 1:
        print("Erreur : N doit etre >= 1")
    else:
        nombres = input("Entrez les nombres : ").split()

        if len(nombres) != N:
            print("Erreur : le nombre d'elements ne correspond pas a N")
        else:
            # Conversion en entiers
            tableau = []
            for n in nombres:
                tableau.append(int(n))

            max_val = tableau[0]

            for i in range(1, N):
                if tableau[i] > max_val:
                    max_val = tableau[i]

            print("Le maximum est :", max_val)

except ValueError:
    print("Erreur : veuillez entrer uniquement des nombres entiers.")

File: solutions/jose-rivas/java/p-04-loops/exercise-01.java
import java.util.Scanner;

public class SumNumbers {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        try {
            System.out.print("Entrez un nombre N : ");
            int N = scanner.nextInt();

            if (N < 1) {
                System.out.println("Erreur : N doit etre >= 1");
                return;
            }

            int somme = 0;

            for (int i = 1; i <= N; i++) {
                somme = somme + i;
            }

            System.out.println("La somme de 1 a " + N + " est : " + somme);

        } catch (Exception e) {
            System.out.println("Erreur : veuillez entrer un nombre entier.");
        }

        scanner.close();
    }
}